src/api/services/base.py

```diff
@@ -7,19 +10,25 @@ class ContentService:
     def __init__(self, repository: any = Depends()):
```

Comment:
У нас есть базовый класс ContentRepositry - давай добавим его в аннотации. 
```suggestion
    def __init__(self, repository: ContentRepository= Depends()):
```
Также - сделай этот класс абстрактным. Есть вероятность, что `Depends` не будет наследоваться, в таком случае, убери его от сюда и вставь в наследуемые классы (`TaskService` и `CategoryService`)